drapps/README.md

```diff
@@ -0,0 +1,102 @@
+# DRApps
+
+## About 
+
+DRApps is simple CLI tool for creating your custom application in Data Robot cloud.
+
+Custom application it small web server hosted by Data Robot in Kubernetes environment.
+Data Robot do not provide any storage resources for this server, but you can use Data
+Robot API and other Internet services.
+
+## Installation
+
+For using this CLI script you need only install python requirements with:
+
+    pip install -r requirements.txt
+
+## Usage
+
+    $ ./drapps.py --help
+    Usage: drapps.py [OPTIONS]
+    
+      App that uses local file for create new custom application
+    
+    Options:
+      -e, --base-env TEXT   Name or ID for execution environment  [required]
+      -p, --path DIRECTORY  Path to folder with files that should be uploaded
+                            [required]
+      -n, --name TEXT       Name for new custom application. Default: CustomApp
+      -t, --token TEXT      Pubic API access token.
+      -E, --endpoint TEXT   Data Robot Public API endpoint
+      --help                Show this message and exit.
+
+### --base-env
+
+UUID or name of Execution Environment that will be used as base for your app setup.
+Execution Environment contains preinstalled essential libraries and packages, that 
+can be used by your application.
+
+You can find list of available Execution Environments in **Custom Model Workshop**
+on [**Environments page**](https://app.datarobot.com/model-registry/custom-environments)
+
+### --path
+
+Path to folder which will be used to create custom application. Files from this folder
+will be uploaded to Data Robot and used for creating custom application image. Custom
+application will be started from this image.
+
+### --name
+
+Name for your custom application. Default value is `CustomApp`. This name will be also
+used for generating name for custom application image by adding `Image` suffix.
```

Comment:
Why does the user need to know what name their image has?

Reply:
There is no specific need in this info. But because customer have access to custom model image API, i wanted to make they know, what comes from where.